TFLite/src/main.py

```diff
@@ -1,21 +1,46 @@
-import TFLiteParser
-import os
-from common import Storage
+""" main module to run inference or load data into database
+
+Contains functions to load model data into database or run inference 
+and print it.
+
+CLA to module and Args to all functions in module:
+    filename (str): name of the file to be parsed. Must be present in 
+        models directory.
+    model_name (str): unique model name of the model being parsed.
+    category (str): problem category of the model.
+"""
+
 import argparse
+import os
 
-import timeit
+import TFLiteParser
+from common import Storage
 
-# Function to load data in models/filename to spanner DB
 def load_data(filename, model_name, category):
```

Comment:
Take the Spanner instance and db as args?

Reply:
Spanner instance and db name are arguments to Storage class, constant strings in load_data() in main.py files.